conjur/api/client.py

```diff
@@ -105,19 +105,19 @@ def __init__(self,
         if api_key:
             logging.debug("Using API key from parameters...")
             self._api = Api(api_key=api_key,
-                            http_debug=http_debug,
-                            login_id=login_id,
-                            ssl_verify=ssl_verify,
-                            **loaded_config)
+                                  http_debug=http_debug,
+                                  login_id=login_id,
+                                  ssl_verify=ssl_verify,
+                                  **loaded_config)
         elif password:
             logging.debug("Creating API key with login ID/password combo...")
             self._api = Api(http_debug=http_debug,
-                            ssl_verify=ssl_verify,
-                            **loaded_config)
+                                  ssl_verify=ssl_verify,
```

Comment:
why this indentation? Should be inline with the above params

Reply:
Fixed